analyzer/src/funTest/assets/projects/synthetic/pub/flutter-project-with-android-and-cocoapods/pubspec.lock

```diff
@@ -7,7 +7,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.6.1"
+    version: "2.8.2"
```

Comment:
> As a side-effect, this fixes an issue where IntelliJ cannot show the
diff of the test

So what was the root cause of this, simple the big size of the expected result?

Reply:
That's my guess, but I can't tell for sure.

Reply:
> @sschuberth After this is merged we could retry #5418.

Do you also have an educated guess *why* this PR could help to avoid getting different test results in- and outside of Docker? I really had no clue when looking at it what should make a difference there.

Reply:
I manually checked why the test was failing for me locally and the reason was that the revision of some flutter dependencies has changed which is now updated in the new expected result. Without knowing why the test failed in Docker I was just hoping it might be the same reason. Could it be that the test was just not executed on CI for a long time due to caching?

Reply:
> Could it be that the test was just not executed on CI for a long time due to caching?

I don't think so... I had these mismatches of test results in- and outside of Docker right from the start, when I introduced GitHub actions for these tests. But anyway, I'll certainly give my #5418 a try again after the merge.

Reply:
Still, #5418 passes now! Great, thanks!